core/src/test/java/com/linecorp/armeria/client/endpoint/dns/DnsNameEncoder.java

```diff
@@ -18,9 +18,9 @@
 import io.netty.buffer.ByteBuf;
 import io.netty.handler.codec.dns.DefaultDnsRecordEncoder;
 
-final class DnsNameEncoder {
+public final class DnsNameEncoder {
```

Comment:
Let's move this class to a test package. 😄

Reply:
There are classes that depends on this class in `armeria.client.endpoint.dns`, so moving this class will not help much.

Reply:
Oops I thought this was in core/src/main....
Please forget about the comment. 😉